Description: set detectJavaApiLink default value to false
Author: tony mancill <tmancill@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=919798
Comment: Also see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=919831
Forwarded: not-needed

--- a/src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java
+++ b/src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java
@@ -587,7 +587,7 @@
      * <br/>
      * See
      * <a href="./apidocs/org/apache/maven/plugin/javadoc/AbstractJavadocMojo.html#DEFAULT_JAVA_API_LINKS">Javadoc</a>
-     * for the default values.
+     * for the default values.  For Debian, the default value is <code>false</code>.  See <a href="https://bugs.debian.org/919798">Debian bug 919798</a>.
      * <br/>
      *
      * @see #links
@@ -595,7 +595,7 @@
      * @see #DEFAULT_JAVA_API_LINKS
      * @since 2.6
      */
-    @Parameter( property = "detectJavaApiLink", defaultValue = "true" )
+    @Parameter( property = "detectJavaApiLink", defaultValue = "false" )
     private boolean detectJavaApiLink;
 
     /**
